### PR TITLE
Added Per-User Permissions

### PR DESCRIPTION
**Pull request by Qw-in**
Added a new USERS tab to the admin page where per-user settings are available. USERS tab is enabled under >Authentication>Enable user permissions. These permissions are stored in a new schema that is updated at login and when the USERS tab is accessed.

I have tested what I can but am very new to meteor and I am sure there are things I have overlooked.

Here is the new settings option:
auth

And here is the USERS tab:
users

### CODE REVIEW

File: lib/collections/publish.js [hunk 1]
```diff
@@ -18,5 +22,9 @@ if (Meteor.isServer) {
 			return TV.find({}, {fields: {user: 0}});
 		}
 	});
-
+	Permissions.allow({
```
Comment (Qw-in) [inline lib/collections/publish.js]: I am unsure if this is a security issue.

File: lib/collections/publish.js [hunk 2]
```diff
@@ -18,5 +22,9 @@ if (Meteor.isServer) {
...
+    	update: function () {
+      		return true;
+		}
+  	});
 };
```
Comment (Qw-in) [inline lib/collections/publish.js]: I am unsure if this is a security issue

Comment (lokenx) [inline lib/collections/publish.js]: Are you updating the Permissions collection client side versus using something like autoform/calling a server side method?

**Comment by lokenx** [comment]
I think one thing needs to be added: some form of notification when you make changes to the users permissions? A simple Bert popup (like everywhere else in the app) or an Update Settings button plus a Bert popup works too.

**Comment by Qw-in** [comment]
Ok, I'll look into adding that. I suppose it would be a bit weird to have just one section without an update button. May try to cleanup the ui while I'm at it.

File: lib/collections/publish.js (hunk 2 above)
Comment (Qw-in) [inline lib/collections/publish.js]: I think adding an update button will remove the need for this code altogether.

**Comment by Qw-in** [comment]
Will I have to create a new pull request or can I just update this one? Sorry, new to git.

**Comment by Qw-in** [comment]
Was still a bit worried about security, so I messed around in the console. I could not access or update the Permissions schema anywhere I couldn't do the same with the Settings schema. For both it was only possible logged in on the admin page so I assume that is ok.

**Comment by lokenx** [comment]
Can't think of anything missing, @rigrassm ?

**Comment by rigrassm** [comment]
I haven't tested it yet but everything looks good to me.

**Comment by lokenx** [comment]
Will generate a release, just having issues with my commits not showing up on GitHub so I can tag it

**Pull request merged**